### DISCUSSION TITLE
wpml issue

### DISCUSSION THREAD

**Discussion by comsup** [flyntwp/flynt#523] [Q&A] [answered]
Hi there,
we have created a website with flynt, all the contents have been put in one language and now we want to create the content in another langauge. We use the "duplicate content" button of WPML on our page but components on the new language are not copied. Did we missed something? we change WPML configuration many times with different options but nothing appears when we duplicate a page. Is there something to do to allow this to work?

**Comment by steffenbew** [comment]
Hi @comsup, that's also how we use WPML. The **duplicate** feature should work as intended.

I just made a fresh installation of the following:
* Flynt v2.0.1
* WPML (sitepress-multilingual-cms.4.6.8)
* Advanced Custom Fields Multilingual (acfml.2.0.5)

Then I created a new page, added 2 WYSIWYG components and duplicated the page.

I got a separate page and was able to edit it within WordPress (not the WPML Translation Editor).

Any idea what might be missing or conflicting on your setup?

**Comment by comsup** [comment, marked as answer]
Hi there,
not sure of the issue but after removing WPML and installing it again all works normally.
Thanks for your feedback,
